.make/test.mk

```diff
@@ -187,6 +193,22 @@ test-remote-no-coverage: prebuild-check $(SOURCES)
 test-migration: prebuild-check migration/sqlbindata.go migration/sqlbindata_test.go
 	F8_RESOURCE_DATABASE=1 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_VERBOSITY_FLAG) github.com/fabric8-services/fabric8-wit/migration
 
+.PHONY: test-e2e
+## Runs the end-to-end tests WITHOUT producing coverage files for each package.
+test-e2e: prebuild-check deps docker-compose-up $(SOURCES)
+	$(call log-info,"Running tests: $@")
+	# ## Clone the fabric8-test repo
+	@if [ "$(FABRIC8_TEST_DIR)" ]; then \
+		echo "NOT Removing existing fabric8-test dir $(FABRIC8_TEST_DIR)"; \
+		rm -rf $(FABRIC8_TEST_DIR); \
+	fi
+	$(GIT_BIN_NAME) clone --depth=1 $(FABRIC8_TEST_REPO)
+	## Start the WIT
+	F8_LOG_LEVEL=ERROR ./wit+pmcd.sh &
+	## Install e2e test deps and run the tests (TBD)
+	sleep 20 && ./fabric8-test/EE_API_automation/cico_run_EE_tests_wit.sh
```

Comment:
Instead of waiting for 20 seconds you should instead do something like this (taken from `cico_setup.sh`):

```bash
function check_postgres_healthiness(){
  echo "CICO: Waiting for postgresql container to be healthy...";
  while ! docker ps | grep postgres_integration_test | grep -q healthy; do
    printf .;
    sleep 1 ;
  done;
  echo "CICO: postgresql container is HEALTHY!";
}
```